doc/theolive.md

```diff
@@ -0,0 +1,161 @@
+# THEOlive support
+
+High-quality real-time video streaming at scale for providers of sports betting, iGaming and interactive entertainment
+
+For more info, visit [THEOlive website](https://www.theoplayer.com/product/theolive).
+
+## Limitations
+
+* Only Flutter Web is supported.
+* Android and iOS will follow later. (You can check out our dedicated [THEOlive Flutter SDK](https://github.com/THEOplayer/flutter-theolive-sdk))
+
+## THEOlive playback
+THEOlive playback is becoming crucial part of THEOplayer's playback engine instead of being available as a separate SDK.
+
+The WEB support is already added, Android and iOS support as part of the THEOplayer SDK will come later. (Right now you can use the [dedicated THEOlive Flutter SDK](https://github.com/THEOplayer/flutter-theolive-sdk) for Android and iOS support) 
+
+### Setting a THEOlive source
+
+```js
+player.source = SourceDescription(sources: [
+    TheoLiveSource(src: "2vqqekesftg9zuvxu9tdme6kl"),
+]);
+```
+
+Instead of using a remote HTTPS url, you can just specify your `channelID` from the [THEOlive Console](https://console.theo.live/) and you can start the playback.
+
+**NOTE:** your THEOplayer license has to contain the `HESP` feature to make it work. (HESP is the underlying technology of THEOlive).
+
+Once the playback starts, you can listen to playback-specify events on THEOplayer.
+
+#### Additional configuration options on `THEOPlayerConfig` 
+By setting a `THEOliveConfiguration` object on `THEOplayerConfig` you are able to specify extra configuration for THEOlive (e.g. a custom sessionId to follow the requests on the backend)
+```js
+    player = THEOplayer(
+        theoPlayerConfig: THEOplayerConfig(
+          license: PLAYER_LICENSE,
+          theolive: TheoLiveConfiguration(externalSessionId: "mySessionID"),
+        ),
+        onCreate: () {
+          print("player is created, ready to use");
+    
+        });
+```
+### Listening for THEOlive-specific events
+
+```js
```

Comment:
using JS for code snippets, because there is no "dart", or it doesn't seem to work.